### PR TITLE
Define codeflow job

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: scripts/publish-canary.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM 652969937640.dkr.ecr.us-east-1.amazonaws.com/containers/node:v18
```
Comment (bangtoven) [inline scripts/publish-canary.Dockerfile:1]: https://github.com/base-org/onchainsummer.xyz/blob/c54c5cb795aaf087f029860232540878065abb61/Dockerfile#L1

**Review by fan-zhang-sv — APPROVED** [review]

**Review by fan-zhang-sv — APPROVED** [review]

**Review by fan-zhang-sv — APPROVED** [review]

**Review by fan-zhang-sv — APPROVED** [review]

**Review by Attila180 — COMMENTED** [review]
hello, thanks again

**Review by Attila180 — COMMENTED** [review]
thanks

**Review by Attila180 — COMMENTED** [review]
Thx

**Review by Attila180 — COMMENTED** [review]
1

**Review by Attila180 — COMMENTED** [review]
Ok

File: .codeflow.yml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 secure:
```
Comment (Attila180) [inline .codeflow.yml:1]: secure:
  required_reviews: 1
  upstream_repository: coinbase/coinbase-wallet-sdk
build:
  engines:
  - BaldurNode:
      name: package-wallet-sdk
      path: "./scripts/publish-canary.Dockerfile"
      compute_type: BUILD_GENERAL1_SMALL
      context: "./"
      autobuild: false

Comment (Attila180) [inline .codeflow.yml:1]: .codeflow.yml

Comment (Attila180) [inline .codeflow.yml:1]: secure:
  required_reviews: 1
  upstream_repository: coinbase/coinbase-wallet-sdk
build:
  engines:
  - BaldurNode:
      name: package-wallet-sdk
      path: "./scripts/publish-canary.Dockerfile"
      compute_type: BUILD_GENERAL1_SMALL
      context: "./"
      autobuild: false

**Review by Attila180 — COMMENTED** [review]
secure:
  required_reviews: 1
  upstream_repository: coinbase/coinbase-wallet-sdk
build:
  engines:
  - BaldurNode:
      name: package-wallet-sdk
      path: "./scripts/publish-canary.Dockerfile"
      compute_type: BUILD_GENERAL1_SMALL
      context: "./"
      autobuild: false

File: .codeflow.yml (hunk 2 above)
Comment (Attila180) [inline .codeflow.yml:1]: secure:
  required_reviews: 1
  upstream_repository: coinbase/coinbase-wallet-sdk
build:
  engines:
  - BaldurNode:
      name: package-wallet-sdk
      path: "./scripts/publish-canary.Dockerfile"
      compute_type: BUILD_GENERAL1_SMALL
      context: "./"
      autobuild: false

**Pull request merged**